app/models/manageiq/providers/ibm_cloud/vpc/manager_mixin.rb

```diff
@@ -19,11 +19,21 @@ def connect(options = {})
   end
 
   # Same as calling connect.
-  # @param _auth_type [nil] Not used
+  # @param _auth_type [nil]
   # @param options [Hash] Connection options.
-  # @return [ManageIQ::Providers::IbmCloud::CloudTools::Vpc]
+  # @return [ManageIQ::Providers::IbmCloud::CloudTools::Vpc] or [Boolean]
   def verify_credentials(_auth_type = nil, options = {})
-    connect(options).authenticator
+    case _auth_type&.to_sym
+    when :events
+      verify_events_credentials
+    else
+      connect(options).authenticator
+    end
+  end
+
+  def verify_events_credentials
+    service_key = authentication_key("events")
+    connect.resource.controller.collection(:list_resource_keys).find {|key| key.dig(:credentials, :service_key) == service_key}.present?
```

Comment:
Also minor style issue
```suggestion
    connect.resource.controller.collection(:list_resource_keys).find { |key| key.dig(:credentials, :service_key) == service_key }.present?
```

Reply:
Do we not need to pass `options` into connect here like we do up [[here]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/359/files#diff-9c1c1fe8a8cf1522f3f23dad6743d18f6c423658360cd877f66c9cb7ac3a1753R30)?  Granted I don't know what is typically in options at this point

Reply:
@agrare `options` from what I can tell is an empty hash, should we remove it?

Reply:
I think lets pass it in to be consistent with the existing `connect()`